README.md

```diff
@@ -5,24 +5,100 @@ Python bindings for the awesome Openpose library.
 Openpose github page:
 https://github.com/CMU-Perceptual-Computing-Lab/openpose
 
+notice!! PyOpenPose uses OpenCV 3.2 and protobuf 2.6.1
 
-### Building
+If you use other operating systems, you have to make sure the versions of opencv and protobuf are correct.
 
-Clone and build openpose. I assume you will also build caffe inside openpose 3rdparty folder.
-Set an environment variable named OPENPOSE_ROOT pointing to the openpose root folder.
-
-Inside the root folder of PyOpenpose run cmake and build with:
+### Install OpenCV 3.2 in ubuntu 16.04 or 14.04
+you can install dependencies of OpenCV using the following command.
+```
+sudo apt-get install -y build-essential cmake pkg-config
+sudo apt-get install -y libjpeg8-dev libtiff5-dev libjasper-dev libpng12-dev
+sudo apt-get install -y libavcodec-dev libavformat-dev libswscale-dev libv4l-dev
+sudo apt-get install -y libxvidcore-dev libx264-dev
+sudo apt-get install -y libgtk-3-dev
+sudo apt-get install -y libatlas-base-dev gfortran
+sudo apt-get install -y python2.7-dev python3.5-dev
+```
+download the source code
+```
+cd ~
+wget -O opencv.zip https://github.com/opencv/opencv/archive/3.2.0.zip
+wget -O opencv_contrib.zip https://github.com/opencv/opencv_contrib/archive/3.2.0.zip
+unzip opencv.zip
+unzop opencv_contrib.zip
+```
 
-```bash
+compile OpenCV without dnn(or error will occur).
+```
+cd opencv/
 mkdir build
 cd build
-cmake ..
-make
+cmake -D CMAKE_BUILD_TYPE=RELEASE \
+    -D CMAKE_INSTALL_PREFIX=/usr/local \
+    -D INSTALL_PYTHON_EXAMPLES=ON \
+    -D INSTALL_C_EXAMPLES=OFF \
+    -D BUILD_opencv_dnn=OFF \
+    -D OPENCV_EXTRA_MODULES_PATH=~/opencv_contrib/modules \
+    -D PYTHON_EXECUTABLE=/usr/bin/python2.7 \
+    -D BUILD_EXAMPLES=ON ..
+```
+install opencv
 ```
+sudo make install
+sudo ldconfig
+```
+you can also uninstall opencv using `sudo make uninstall`
+
+### Installation of protobuf in ubuntu 16.04 or 14.04
```

Comment:
protobuf is installed during openpose installation. It should not be covered here.